siege_utilities/geo/providers/census_geocoder.py

```
@@ -0,0 +1,397 @@
+            addr.get("state", ""),
+            addr.get("zipcode", ""),
+        ])
+
+    # Write to temp file for censusgeocode
+    with tempfile.NamedTemporaryFile(mode="w", suffix=".csv", delete=False) as f:
+        f.write(csv_buffer.getvalue())
+        csv_path = f.name
+
+    try:
+        result = cg.addressbatch(csv_path, returntype="geographies")
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)